modules/tinymce/src/plugins/lists/main/ts/actions/Update.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Obj } from '@ephox/katamari';
+import Editor from 'tinymce/core/api/Editor';
+import { getParentList } from '../core/Selection';
+
+interface ListUpdate {
+  attrs?: Record<string, string>;
+}
+
+export const updateList = (editor: Editor, update: ListUpdate) => {
+  const parentList = getParentList(editor);
+  editor.undoManager.transact(() => {
+    if (update.attrs) {
```

Comment:
Should we do an `isObject` check here?

Reply:
Indeed! I added a test for passing null `update` and it fails. Good catch!